src/panel/add_card/AddCard.js

```diff
@@ -10,13 +10,13 @@ const AddNewCard = props => (
 		<div>
 			{/* TODO: extract component, maintaing spacing */}
 			<label className="pt-label pt-inline">
-				Title  {/* 2 spaces after 'Title' */}
-				<input className="pt-input" style={{width: 200}} type="text" 
+				Title&nbsp;&nbsp;&nbsp;{/* 3 spaces after 'Title' */}

```

Comment:
Using `&nbsp;` for spaces, methinks makes it more clear... thoughts?
Also, if we decide to use this, then we can remove the comment after it